### PR TITLE
Add support for Mojang mappings

### PR DESCRIPTION
**Pull request by Dominilk01**
Yea i know its not the best code, but well it worked to me with mojang mapping

### CODE REVIEW

File: src/main/java/fr/mrmicky/fastboard/FastBoardBase.java [hunk 1]
```diff
@@ -84,23 +84,31 @@ public abstract class FastBoardBase<T> {
...
             Class<?> craftPlayerClass = FastReflection.obcClass("entity.CraftPlayer");
-            Class<?> entityPlayerClass = FastReflection.nmsClass("server.level", "EntityPlayer");
-            Class<?> playerConnectionClass = FastReflection.nmsClass("server.network", "PlayerConnection");
-            Class<?> packetClass = FastReflection.nmsClass("network.protocol", "Packet");
-            Class<?> packetSbObjClass = FastReflection.nmsClass(gameProtocolPackage, "PacketPlayOutScoreboardObjective");
-            Class<?> packetSbDisplayObjClass = FastReflection.nmsClass(gameProtocolPackage, "PacketPlayOutScoreboardDisplayObjective");
-            Class<?> packetSbScoreClass = FastReflection.nmsClass(gameProtocolPackage, "PacketPlayOutScoreboardScore");
-            Class<?> packetSbTeamClass = FastReflection.nmsClass(gameProtocolPackage, "PacketPlayOutScoreboardTeam");
+            Class<?> entityPlayerClass = FastReflection.nmsOptionalClass("server.level", "EntityPlayer")
+                    .orElse(FastReflection.nmsClass("server.level", "ServerPlayer"));
```
Comment (syldium) [inline src/main/java/fr/mrmicky/fastboard/FastBoardBase.java:100]: The Mojang mapped class name will always be looked for, even if `EntityPlayer` is found. And on Spigot servers, `ServerPlayer` doesn't exist, so this code fails.

Unfortunately, making the call lazy with [`Optional#orElseGet`](https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/util/Optional.html#orElseGet(java.util.function.Supplier)) would make the code quite verbose and not ideal:
```java
Class<?> entityPlayerClass = FastReflection.nmsOptionalClass("server.level", "EntityPlayer")
      .orElseGet(() -> {
          try {
              return FastReflection.nmsClass("server.level", "ServerPlayer");
          } catch (ClassNotFoundException ex) {
              throw new RuntimeException(ex);
          }
      });
```

I *think* `FastReflection#nmsClass` could be updated to allow multiple class names to be searched and could (ab)use the fact that both mappings use the same package names:
```java
Class<?> entityPlayerClass = FastReflection.nmsClass("server.level", "EntityPlayer", "ServerPlayer");
```

**Review by syldium — CHANGES_REQUESTED** [review]

**Comment by Dominilk01** [comment]
I thinker that if value not found then it will call the orElse. Il check it tomorrow and update it

**Comment by Dominilk01** [comment]
Don't mind me [syldium](https://github.com/syldium) i stole your idea of a method that accept both ;)

**Comment by MrMicky-FR** [comment]
I pushed changes in `FastReflection` to avoid throwing/catching a dozen of `ClassNotFoundException`, tested and seems to work fine on all versions, but feel free to review before I merge this 😃

**Review by syldium — APPROVED** [review]
Tested on Spigot 1.8.8, Paper+Bukkit plugin 1.21, Paper+non remapped Bukkit plugin 1.21 and Paper dev server 1.21.

**Pull request merged**